Sources/ArcGISToolkit/Components/UtilityNetworkTrace/UtilityNetworkTrace.swift

```diff
@@ -635,9 +642,9 @@ public struct UtilityNetworkTrace: View {
     // MARK: Computed Properties
     
     /// Indicates the number of the trace currently being viewed out the total number of traces.
-    private var currentTraceLabel: String {
+    private var currentTraceLabel: LocalizedStringKey {
         guard let index = viewModel.selectedTraceIndex else { return "Error" }
-        return "Trace \(index+1) of \(viewModel.completedTraces.count.description)"
+        return "Trace \(index+1) of \(viewModel.completedTraces.count)"
```

Comment:
It may not look like it, but these values are now formatted (rather than the unformatted `description` which string interpolation would normally produce). Because the return type is `LocalizedStringKey`, [this](https://developer.apple.com/documentation/swiftui/localizedstringkey/stringinterpolation/appendinterpolation(_:)-378v9) interpolation method is now being used.